india_compliance/public/js/components/data_table_manager.js

```
@@ -144,4 +147,134 @@
+            if (cell.content === "") return cell;
+
+            const fieldname = cell.column.id;
+
+            if (row.label_column === fieldname) {
+                cell.content = row.label;
+            } else if (
+                cell.column._fieldtype === "Float" ||
+                cell.column.fieldtype === "Float"
+            ) {
+                cell.content = data[fieldname] || 0.0;
+            } else if (data.hasOwnProperty(fieldname)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)